client/src/views/Dashboard/Dashboard.js

```diff
@@ -61,26 +57,21 @@ const Dashboard = props => {
   useEffect(() => {
     dispatch(loadAllPeerForUser(currentUser._id))
     dispatch(loadAllSelfReviewsForUser(currentUser._id))
-  }, [dispatch])
+  }, [currentUser._id, dispatch])
 
   const peerReviewsArray = []
   let filteredEmployee
   if (peerReviews) {
     filteredEmployee = peerReviews.filter(
       cls =>
-        cls.employee_under_review.firstname +
-          ' ' +
-          cls.employee_under_review.lastname
-            .toLowerCase()
-            .includes(searchText.toLowerCase().trim()) &&
         cls.status !== 'Done' &&
         cls.status !== 'Inactive'
     )
     filteredEmployee.map((review, key) => {
       peerReviewsArray.push([
         review.employee_under_review.firstname +
-          ' ' +
-          review.employee_under_review.lastname,
+        ' ' +
+        review.employee_under_review.lastname,
```

Comment:
I think we need to use template literal here. Is it possible to use template literal?

Reply:
did in PR 31